guest-tools/direct-boot/README.md

```diff
@@ -0,0 +1,123 @@
+# TD Boot methods 
+
+This folder contains scripts and instructions to boot a TD using the [direct
+boot](https://qemu-project.gitlab.io/qemu/system/linuxboot.html) method.
+
+The direct boot method is an alternative to the boot method that is being used by `tdvirsh`
+to run TDs.
+We will refer to the `tdvirsh` boot method as the `indirect boot` method.
+With the `indirect boot` method, the boot chain involves following components:
+- TDVF (virtual UEFI firmware)
+- SHIM
+- Grub
+- Kernel + Initrd
+
+The `direct boot` will skip `SHIM` and `Grub` in the boot chain by providing the `Kernel`
+and `Initrd` directly to `qemu`. Per consequence, the boot chain involves these components:
+- TDVF (virtual UEFI firmware)
+- Kernel + Initrd
+
+The boot chain structure is important for remote attestation since it impacts the size of
+the event log journal. Indeed, each component in the boot chain generates a set of entries of the event
+log journal. The more components we have in the boot chain, the more event logs we will have and the harder
+is the verification of the correctness of the measurement values.	
+
+### Prerequisites
+
+We need to generate different files used by direct boot scripts:
+
+NOTE : the following instructions are for `24.04` guest but please replace it by `24.10` if
+you want to work with `oracular` TD guest.
+
+```
+$ cd guest-tools/image
+$ ./create-td-uki.sh tdx-guest-ubuntu-24.04-generic.qcow2
+```
+
+This script will generate 3 files:
+- `vmlinuz-24.04` : the kernel of the guest image
+- `initrd.img-24.04` : the initrd of the guest image
+- `uki.efi-24.04` : the Unified Kernel Image that bundles together the kernel and the initrd
+
+### Direct boot
+
+```
+$ cd guest-tools/direct-boot
+$ ./boot_direct.sh 24.04
+```
+

```

Comment:
Suggest adding a link to the main README here where we provide the login credentials.

Reply:
I agreed and added a note in the prerequisite, can you check it out and tell me whether it is enough ? I m fine if you want to have some thing at this location so that users can catch it more easily

Reply:
My bad, I missed that